File: app/src/main/java/com/artmurka/artmurkaapp/Model/Pojo/ItemList/ItemBasket/Summ.java

package com.artmurka.artmurkaapp.Model.Pojo.ItemList.ItemBasket;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Summ {

    @SerializedName("summ")
    @Expose
    private String summ;
    @SerializedName("summ_raw")
    @Expose
    private String summRaw;

    public String getSumm() {
        return summ;
    }

    public void setSumm(String summ) {
        this.summ = summ;
    }

    public String getSummRaw() {
        return summRaw;
    }

    public void setSummRaw(String summRaw) {
        this.summRaw = summRaw;
    }

}

File: app/src/main/java/com/artmurka/artmurkaapp/data/model/pojo/itemlist/itembasket/Summ.kt
package com.artmurka.artmurkaapp.data.model.pojo.itemlist.itembasket

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Summ {

    @SerializedName("summ")
    @Expose
    var summ: String? = null
    @SerializedName("summ_raw")
    @Expose
    var summRaw: String? = null

}
